jupyterhub/tests/test_services.py

```diff
@@ -89,10 +89,10 @@ async def test_external_service(app):
             }
         ]
         await maybe_future(app.init_services())
-        await maybe_future(app.init_roles())
+        await maybe_future(app.init_role_creation())
         await app.init_api_tokens()
         await app.proxy.add_all_services(app._service_map)
-        await app.init_roles()
+        await app.init_role_creation()
```

Comment:
```suggestion
        await app.init_role_assignment()
```

Reply:
Thanks for catching that

Reply:
Did you commit a change? I still see creation here.

Reply:
Slipped through, fixed in the next commit, thanks!